thrift/shared.thrift

```diff
@@ -1969,3 +1969,23 @@ struct AsyncWorkflowConfiguration {
   // queueConfig is the configuration for the queue if predefined_queue_name is not used
   40: optional DataBlob queueConfig
 }
+
+/**
+* Any is a logical duplicate of google.protobuf.Any.
+*
+* The intent of the type is the same, but it is not intended to be directly
+* compatible with google.protobuf.Any or any Thrift equivalent - this blob is
+* RPC-type agnostic by design (as the underlying data may be transported over
+* proto or thrift), and the data-bytes may be in any encoding.
+*
+* See TypeID to interpret the contents.
+**/
+struct Any {
+  // Type-string describing value's contents, and intentionally avoiding the
+  // name "type" as it is often a special term.
+  // This should be a hard-coded string of some kind.
+  10: optional string TypeID
```

Comment:
ID suffix might mislead unless we repeat the above comment everywhere. maybe rename to `ValueType`

Reply:
true, we do use "id" to mean "enum" a lot...
I'll go change that 👍